donkeycar/pipeline/training.py

```diff
@@ -76,8 +76,8 @@ def create_tf_data(self) -> tf.data.Dataset:
         return dataset.repeat().batch(self.batch_size)
 
 
-def train(cfg: Config, tub_paths: str, model: str, model_type: str) \
-        -> tf.keras.callbacks.History:
+def train(cfg: Config, tub_paths: str, model: str, model_type: str,
+          select: Optional[str] = None) -> tf.keras.callbacks.History:
```

Comment:
This should not be a string at all. It should be a function or a lambda.

Reply:
Yeah, see solution above.